assets/csidriveroperators/aws-ebs/hypershift/guest/kustomization.yaml

```diff
@@ -17,3 +17,9 @@ patches:
         name: PLACEHOLDER
     target:
       annotationSelector: "storage.openshift.io/remove-from=guest"
+  # remove these annotations as they're just noise post-kustomization
+  - target:
+      annotationSelector: "storage.openshift.io/remove-from=mgmt"
+    patch: |
+      - op: "remove"
+        path: "/metadata/annotations/storage.openshift.io~1remove-from"
```

Comment:
Can you add a comment here that `~1` is being used because, using `/` will not work for JSON path  and hence `~1` is replaced version of `/`'? And lets do this for all the removals.

Reply:
Done